src/OrchardCore.Modules/OrchardCore.Autoroute/Handlers/AutorouteContentLocalizationHandler.cs

```diff
@@ -0,0 +1,21 @@
+using System.Threading.Tasks;
+using OrchardCore.Autoroute.Models;
+using OrchardCore.ContentManagement;
+
+namespace OrchardCore.ContentLocalization.Handlers;
+
+public class AutorouteContentLocalizationHandler : IContentLocalizationHandler
```

Comment:
```suggestion
internal sealed class AutorouteContentLocalizationHandler : IContentLocalizationHandler
```
No need to add this as a public class. IMHO we should in general avoid to increase the API surface, if there is no specific reason.